From 5710038c33435881a578a0602cfe27ac5bc4c781 Mon Sep 17 00:00:00 2001
From: Luna D Dragon <luna.dragon@suse.com>
Date: Mon, 15 Sep 2025 15:30:15 +0530
Subject: [PATCH] Fix: hostadd allow device form to overflow on X-axis

If user has a host with large vendor ids, or pci-e ids the hostadd card
may overflow and be cut off for example on a host with > 100 pci-e
lanes. This adds a div with overflowX set to auto so if there is a
overflow it can overflow and add a scroll bar
---
 src/components/vm/hostdevs/hostDevAdd.tsx | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/components/vm/hostdevs/hostDevAdd.tsx b/src/components/vm/hostdevs/hostDevAdd.tsx
index 6a88e59a..8ed08f42 100644
--- a/src/components/vm/hostdevs/hostDevAdd.tsx
+++ b/src/components/vm/hostdevs/hostDevAdd.tsx
@@ -251,7 +251,9 @@ const AddHostDev = ({
     const body = (
         <Form isHorizontal>
             <TypeRow type={type} setType={setTypeWrapper} />
-            <DevRow idPrefix={idPrefix} selectableDevices={selectableDevices} setSelectableDevices={setSelectableDevices} />
+            <div style={{ overflowX: "auto" }}>
+                <DevRow idPrefix={idPrefix} selectableDevices={selectableDevices} setSelectableDevices={setSelectableDevices} />
+            </div>
         </Form>
     );
 
-- 
2.51.0

